modules/machine-health-checks-about.adoc

```diff
@@ -6,15 +6,32 @@
 [id="machine-health-checks-about_{context}"]
 = About machine health checks
 
-Machine health checks automatically repair unhealthy machines in a particular machine pool.
+You can define conditions under which machines in a Cluster are considered unhealthy by using a `MachineHealthCheck` resource.
```

Comment:
@JoelSpeed please can you change "Cluster" to "cluster"? Sorry I keep spotting new items when I run through the final review.